scripts/test

```diff
@@ -10,4 +10,4 @@ echo "running unit tests"
 go test -p 1 -v ./pkg/simulator/...
 
 echo "running integration tests"
-go test -v ./tests/integration
\ No newline at end of file
+go test -v -timeout=20m ./tests/integration
```

Comment:
I thought the new test would be good. But could we add some variable to skip the test SB loading to increase the local build?

Reply:
i agree we could skip the integration tests via a variable and only use the integration tests to run for tagged builds.